picasso-layoutlib-compat/api/picasso-layoutlib-compat.api

```diff
@@ -0,0 +1,5 @@
+public final class com/squareup/picasso3/LayoutLibThreadFactory : java/util/concurrent/ThreadFactory {
```

Comment:
Let's chat the module package structure and update the Factory name (lowercased l in Lib)
```suggestion
public final class com/squareup/picasso3/layoutlib/LayoutlibThreadFactory : java/util/concurrent/ThreadFactory {
```